### DISCUSSION TITLE
[Solved]  THERE ARE NO INPUTS TO FOCUS ON

### DISCUSSION THREAD

**Discussion by theomkumar** [gdh1995/vimium-c#746] [Q&A] [answered]
In Normal Mode I can scroll through the code and even search keywords in the code, but 'gi' is unable to focus back on the code.
"THERE ARE NO INPUTS TO FOCUS ON"
I tried with Vimium, it works there.

EDIT :  This fixed this issue!
`run i focusInput:(<c-m>:W+150wait)%cfocusInput \
  o.keep o.select="all-line" o.reachable=false \
  o.clickable=".js-toggle-inline-comment-form" \
  o.prefer="#js-issues-search,#searchEngines"`

**Comment by gdh1995** [comment, marked as answer]
The complex mapping of `run i ...`  will:
1. try `focusInput`
2. when failing, go to the most top frame (this means an outer page area) or scroll to top
3. do `focusInput` again

And those `o.xxx=xxx` is to make `focusInput` find elements better. `focusInput` should have found all visible common input boxes without them.

So I'm still curious why Vimium can find something but Vimium C's simple `focusInput` can not. Could you provide a test URL and some screen snapshots?
